### ISSUE TITLE
Config split recipe

### ISSUE THREAD

**Issue by andycarlberg** [acquia/drupal-recommended-settings#54]
**Is your feature request related to a problem? Please describe.**
The recommended settings will handle config splits but expect a certain naming convention. This is not documented well (yet). If you make your own splits but don't follow or aren't aware of this convention, the splits fail to apply correctly. 

In BLT, this was handled via a recipe that would generate the split config and the matching config directories so you could be sure your splits matched what the settings expected.

**Describe the solution you'd like**
I would suggest that this package implement a Drush command or Composer command to generate the splits according to a recipe. This seems like the appropriate place to implement it because it is specific to this package. If I'm not using this package to manage settings, then I don't need to follow this convention.

At a minimum, I think this package needs to document the expected config split convention.

**Describe alternatives you've considered**
This was previously handled in BLT but that project is EOL. Documentation would probably suffice but something more automated would be better.

**Comment by vishalkhode1** [comment]
Hi @andycarlberg
Can you please confirm, are you expecting that DRS should provide a recipes `blt recipes:config:init:splits` and/or other recipes as well (or a documentation on how to create a new plugin/drush command for the same) ?
I can create a new plugin/drush command and but it would be great, if you could help us in testing/reporting bugs (If any).

**Comment by andycarlberg** [comment]
Yes, that's correct. The `blt recipes:config:init:splits` command is what I feel should be replicated. It would help developers setup their config splits in the way this project expects so that it can handle automatic loading.

I would be happy to help test and report bugs.
